crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -524,6 +527,7 @@ impl<'a> ExecContextFactory<'a> {
             task_hash,
             execution_env,
             continue_on_error: self.visitor.opts.run_opts.continue_on_error,
+            pass_through_args,
```

Comment:
Nit we can avoid passing this into the constructor and use the factory to build it instead:
```suggestion
            pass_through_args: self.visitor.opts.run_opts.args_for_task(&task_id),
```
(This might need to be hoisted to happen before `task_id` gets moved in another part of creating the `ExecContext`)

Reply:
Good call, moved to factory.